src/bin/checkrad.in

```diff
@@ -1293,7 +1293,7 @@ sub mikrotik_telnet {
   ($login, $password) = naspasswd($terminalserver, 1);
   return 2 if ($password eq "");
 
-  # MikroTik routeros doesnt tell us to which port the user is connected
+  # MikroTik router doesn't tell us to which port the user is connected
```

Comment:
Incorrect amendment.  Should be RouterOS.

Reply:
Fixed.